templates/deno/src/models.d.ts.twig

```diff
@@ -6,6 +6,8 @@
         {% else %}
             {{property.sub_schema | caseUcfirst}}{% if property.type == 'array' %}[]{% endif %}
         {% endif %}
+    {% elseif property.sub_schemas %}
+        {% for subProperty in property.sub_schemas %}{{subProperty | caseUcfirst}}{% if property.type == 'array' %}[]{% endif %}{% if loop.last == false %} | {% endif %}{% endfor %}
```

Comment:
This will not work 👍🏻  it needs to be `(Type | Type)[]`

Reply:
Updates:

```ts
        /**
         * Collection attributes.
         */
        attributes: (AttributeBoolean | AttributeInteger | AttributeFloat | AttributeEmail | AttributeEnum | AttributeUrl | AttributeIp | AttributeString)[];
```